docs/markdown/Machine-files.md

```diff
@@ -155,6 +153,13 @@ case). For compilers like MSVC and Clang-Cl, this is the path to a
 linker for Meson to invoke, such as `link.exe` or `lld-link.exe`.
 Support for `ld` is *new in 0.53.0*
 
+These values must be either strings or an array of strings. Previous to Meson
+1.3.0, if a path contains spaces, it must be wrapped into an array:
```

Comment:
Perhaps this would be clearer worded like:

```
These values must be either strings or an array of strings. Prior to
Meson 1.3.0, if a path contained spaces, it has to be wrapped into an
array:
```

Thanks!